news.rst

```diff
@@ -1,6 +1,19 @@
 Release Notes
 ========================================
 
+Version 3.6.1, Not Yet Released
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Notice: Botan 3.7.0 will remove support for the currently supported
+  experimental Kyber r3 TLS ciphersuites, leaving only the standardized
+  ML-KEM suites. (GH #4403)
```

Comment:
Yeah, February should be enough grace period for that. IMO: These things were added to enable easy-to-use experimentation with the new groups. Now that we have the standard, it should be fine to just remove those.